core/tests/protractor/accessibility.js

```diff
@@ -41,6 +88,24 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test navigation shortcuts for static webpages',
+    async function() {
+      // Should create a user and login.
+      await users.createUser('user11@accessibility.com', 'user11accessibility');
+      await users.login('user11@accessibility.com', true);
+
+      // Should test navigation shortcuts.
+      await testNavigationShortcuts('community-library');
+      await testNavigationShortcuts('creator-dashboard');
+      await testNavigationShortcuts('get-started');
+      await testNavigationShortcuts('about');
+      await testNavigationShortcuts('privacy-policy');
+      await testNavigationShortcuts('donate');
+      await testNavigationShortcuts('preferences');
+      await testNavigationShortcuts('learner-dashboard');
+      await testNavigationShortcuts('notifications');
```

Comment:
I know that you're trying to minimize code duplication, but in tests, it's okay to be a bit redundant for clarity. Let's have a separate test case for each page and not hide the actual test details in a helper function. The reader should be able to look at a test case and understand what is happening without looking at the helper functions.

Reply:
Got it, made some unique tests